packages/calling-component-bindings/src/hooks/usePropsFor.ts

```diff
@@ -115,6 +121,8 @@ export type GetSelector<Component extends (props: any) => JSX.Element | undefine
   ? /* @conditional-compile-remove(PSTN-calls) */ HoldButtonSelector
   : AreEqual<Component, typeof _CaptionsBanner> extends true
   ? _CaptionsSelector
+  : AreEqual<Component, typeof _StartCaptionsButton> extends true
```

Comment:
Same here, these types require to be exported, so this one might cause a type leak